search/support.adoc

```diff
@@ -5,12 +5,14 @@
 [[supported-versions]]
 == Supported Products pass:[<i class="icon-user-md icon-fixed-width icon-2x"></i>]
 
-Hibernate Search is available as an enterprise ready supported library as a component of the http://www.redhat.com/en/technologies/jboss-middleware/application-platform[Red Hat JBoss Enterprise Application Platform].
+Hibernate Search is a fully supported component of the https://www.redhat.com/en/products/runtimes[Red Hat Runtimes] offering.
 
-In addition the http://www.redhat.com/en/technologies/jboss-middleware/data-grid[Red Hat JBoss Data Grid] provides optional extensions such as high performance real time replication for Hibernate Search powered applications running on the Red Hat JBoss Enterprise Application Platform.
+Using JBoss EAP, Quarkus, Open Liberty or Spring Boot? They are all part of Red Hat Runtimes.
 
-The Red Hat JBoss Data Grid also embeds a custom edition of Hibernate Search and allows using most of the same API to run high performance full-text queries and data mining on the data grid.
+In addition, the https://www.redhat.com/en/technologies/jboss-middleware/data-grid[Red Hat Data Grid] provides optional extensions such as high performance real time replication for Hibernate Search powered applications running on one of the Red Hat supported runtime.
+
+The Red Hat Data Grid also embeds a custom edition of Hibernate Search and allows using most of the same API to run high performance full-text queries and data mining on the data grid.
```

Comment:
Not totally happy of how this turned out. @yrodiere could you have a look and adjust as you see fit?

Reply:
They are in the process of removing Hibernate Search APIs from Infinispan, so I'd go with this:

```suggestion
[Red Hat Data Grid](https://www.redhat.com/en/technologies/jboss-middleware/data-grid) also uses a custom edition of Hibernate Search internally to provide high performance full-text queries and data mining on the data grid.
```